aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -549,6 +554,11 @@ Tensor _sparse_csr_mm(const Tensor& mat1, const Tensor& mat2) {
         0.0,
         1.0);
   }
+  if (mat1.layout() == kSparseCsc && mat2.layout() == c10::kStrided) {
+    // TODO: This is a costly conversion. We should have
+    // native support for CSC.
+    return _sparse_csr_mm(mat1.to_sparse_csr(), mat2);
```

Comment:
Or use `return mm(mat2.transpose(0, 1), mat1.as_sparse_csr()).transpose(0, 1)` approach. Can be a follow up.